src/Command/ShopmonCreateAclRole.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Frosh\Tools\Command;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Api\Acl\Role\AclRoleCollection;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\Doctrine\RetryableQuery;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityCollection;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
+use Shopware\Core\Framework\DataAbstractionLayer\Write\WriteException;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\Framework\Validation\WriteConstraintViolationException;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand('frosh:shopmon:create-acl-role', 'Create Shopmon ACL role')]
+class ShopmonCreateAclRole extends Command
+{
+    private const SHOPMON_ACL_ROLE_NAME = 'shopmon_acl_role';
+    private const SHOPMON_ACL_ROLE_DESCRIPTION = 'This role has the necessary permissions to use Shopmon';
+    private const SHOPMON_ACL_ROLE_ID = '018dd6ae4c4072b1b5887fe8d3b9b95a';
+
+    /**
+     * @param EntityRepository<AclRoleCollection> $aclRoleRepository
+     * @param EntityRepository<EntityCollection> $userRepository
+     * @param Connection $connection
+     */
+    public function __construct(
+        private readonly EntityRepository $aclRoleRepository,
+        private readonly EntityRepository $userRepository,
+        private readonly Connection       $connection
+    )
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->addOption('username', null, InputOption::VALUE_OPTIONAL, 'User that will be assigned the role');
```

Comment:
why a user, we should only create a integration? 🤔

Reply:
user:create command doesn't support ACL stuff

so in a CI/CD setup first create a shopmon user with user:create and create+assign the role with this command